spec/std/callstack_spec.cr

```diff
@@ -17,9 +17,9 @@ describe "Backtrace" do
     output = `#{tempfile.path}`
 
     # resolved file line:column
-    output.should match(/#{sample} 3:10/) # callee1
-    output.should match(/#{sample} 15:3/) # callee3
-    output.should match(/#{sample} 17:1/) # ???
+    output.should match(/callee1 at #{sample} 3:10/)
+    output.should match(/callee3 at #{sample} 15:3/)
```

Comment:
Thanks for testing and providing binaries! I'll try to read the sections and dwarf from them —I made a Mach-O and ELF reader that outputs data like `readelf` does. I should publish it somewhere :-)